app/assets/stylesheets/comp-block-header.scss

```diff
@@ -0,0 +1,29 @@
+.gobierto-filter-block-header {
```

Comment:
I noticed you're importing these CSS files only from JS. That's cool!
So, only a thing about the naming, if they're gonna be named `gobierto-WHATEVER`, why is this one called `gobierto-filter-block-header`, instead of `gobierto-block-header`?